### ISSUE TITLE
svelte-repl component breaks site keyboard navigation

### ISSUE THREAD

**Issue by ZerdoX-x** [sveltejs/svelte-repl#163]
https://github.com/sveltejs/svelte-repl/blob/2d0ecdf288c117f93a208efaaf04f2e38240fb86/src/CodeMirror.svelte#L292
```
<textarea
  tabindex='2'
></textarea>
```

Why?...
I guess you want to place spaces in textarea using tab button.
I don't think it does worth it.. 

For example I am writing this text in github textarea. I can easily "jump out" from this element by pressing `tab` button and then navigate to "Submit new issue" button :)

**Comment by dummdidumm** [comment]
Not being able to use the tab key in a code area would be a no-go for me. It definitely should be possible to escape the code area though, maybe through ctrl+tab or by pressing escape.

**Comment by geoffrich** [comment]
So, this is a pretty significant accessibility issue. Having a positive tabindex in the REPL means that it's placed first in the tab order, and is almost always a [significant antipattern](https://dequeuniversity.com/rules/axe/4.2/tabindex). When a keyboard user visits the Svelte homepage and presses Tab, the REPL example halfway down the page is the first thing focused, which they then can't get out of because Tab then triggers indentation. This effectively prevents keyboard-only users from navigating the Svelte homepage.

I can't see why the textarea needs a tabindex at all, since they should be keyboard focusable by default. I might be missing something, since it was clearly placed there intentionally. However, removing this attribute would vastly improve the keyboard accessibility of the page, since everything above the REPLs on the homepage would be navigable with the keyboard before getting trapped in the first example.

Preventing keyboard users from being trapped in the examples would be trickier, since the majority of users want to use Tab in a code area. The next version of CodeMirror will allow [escaping the code area using ESC](https://codemirror.net/6/examples/tab/), so it could make sense to replicate this behavior. I'm not sure how much effort this would take, but it would fix a significant a11y bug.

I'm willing to submit a PR fixing the immediate tabindex issue for a quick win, and circle back to the larger keyboard trap issue later.

Related CodeMirror issue: https://github.com/codemirror/CodeMirror/issues/5444

**Comment by Rich-Harris** [comment]
fixed via https://github.com/sveltejs/sites/pull/122!
